src/main/java/seedu/taassist/model/Model.java

```diff
@@ -108,6 +115,9 @@ public interface Model {
     /** Returns an unmodifiable view of the module class list */
     ObservableList<ModuleClass> getModuleClassList();
 
+    /** Checks if all {@code moduleClass} in {@code moduleClasses} exists */
+    boolean areAllExistingModuleClasses(Collection<ModuleClass> moduleClasses);
```

Comment:
How about naming it `hasModuleClasses`? Since there is `deleteModuleClass` and `deleteModuleClasses`, I think that would be better.